addons/stock_fleet/models/fleet_vehicle_model.py

```diff
@@ -29,3 +29,14 @@ def _compute_weight_capacity_uom_name(self):
 
     def _compute_volume_capacity_uom_name(self):
         self.volume_capacity_uom_name = self.env['product.template']._get_volume_uom_name_from_ir_config_parameter()
+
+
+class FleetVehicle(models.Model):
+    _inherit = 'fleet.vehicle'
+
+    def _compute_display_name(self):
+        if self.env.context.get('from_picking_batch'):
+            for record in self:
```

Comment:
```suggestion
            for vehicle in self:
```
Use `self.env._` instead of `_`.

Also, add `@api.depends_context('from_picking_batch')`, Rest seems ok to me. You can ask for the final review to abdu.